backend-api/tests/api-tests/events.test.ts

```diff
@@ -0,0 +1,157 @@
+import { AxiosInstance } from "axios";
+import { randomUUID } from "crypto";
+import "dotenv/config";
+import {
+  APIS,
+  EVENTS_API_INSTANCE,
+  SESSIONS_API_INSTANCE,
+  STS_MOCK_API_INSTANCE,
+} from "./utils/apiInstance";
+import {
+  ClientDetails,
+  createSessionForSub,
+  CredentialRequestBody,
+  getCredentialAccessToken,
+  getCredentialRequestBody,
+  getFirstRegisteredClient,
+} from "./utils/apiTestHelpers";
+
+const ONE_SECOND = 1000;
+jest.setTimeout(2 * 5 * ONE_SECOND);
```

Comment:
```suggestion
jest.setTimeout(10000);
```

Personally find the sum 2 * 5 * ONE_SECOND a bit confusing but could definitely just be me 😆 

Also why do we need a sum here?

Reply:
I think this started as provisioning 5 seconds for a giving amount of lambdas (in this instance 2) which I think is a bit clearer. But happy to revisit if needed